src/test/run-make/rustdoc-test-builder/Makefile

```diff
@@ -0,0 +1,9 @@
+-include ../../run-make-fulldeps/tools.mk
+
+# How to manually run this
+# $ ./x.py test src/test/run-make/rustdoc-test-builder
+
+all:
+	# See #81197 for why this uses `rustc` instead of `true`
```

Comment:
Please write out the reason in this comment. There's no long discussion or anything that citing a PR would help for.